styles/designs/carnival/parts/_columns-components.scss

```diff
@@ -158,6 +182,22 @@ $Columns--FourColumnsSixRatioOne: (
     )
 );
 
+$Columns--FourColumnsSixRatioOneA: (
+    _name: "FourColumnsSixRatioOne",
+    _subselector: " [data-columns='4'][data-ratio='1,1,2,2'] col[data-width='1*']",
+    _properties: (
+        width: column-width(4,6,1),
+    )
+);
```

Comment:
is the difference between this just the selector?

Reply:
yes, I thought a new component was needed because of the data ratio subselector.